From: Takashi Iwai <tiwai@suse.de>
Subject: drm/nouveau: Don't spew kernel WARNING for each timeout
Patch-mainline: Never, SLE15 specific
References: bsc#1126480

Although the lock up of nouveau GPU was worked around recently, it
still spews the ugly kernel warning with stack trace at each time,
which is quite annoying.

As we do know that it's a problem, let's paper over it.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/gpu/drm/nouveau/include/nvkm/subdev/timer.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/gpu/drm/nouveau/include/nvkm/subdev/timer.h
+++ b/drivers/gpu/drm/nouveau/include/nvkm/subdev/timer.h
@@ -59,7 +59,7 @@ s64 nvkm_timer_wait_test(struct nvkm_tim
 	} while ((_taken = nvkm_timer_wait_test(&_wait)) >= 0);                \
                                                                                \
 	if (_warn && _taken < 0)                                               \
-		dev_WARN(_wait.tmr->subdev.device->dev, "timeout\n");          \
+		dev_info(_wait.tmr->subdev.device->dev, "timeout\n");          \
 	_taken;                                                                \
 })
 #define nvkm_usec(d,u,cond...) nvkm_nsec((d), (u) * 1000, ##cond)
